godtools/Views/Buttons/GTButton.swift

```diff
@@ -79,6 +79,17 @@ class GTButton: UIButton {
         self.layoutSubviews()
     }
     
+    func designAsTractModalButton() {
```

Comment:
Will tract modal buttons be able to be customized or follow the default colors as defined in the manifest?